XamlControlsGallery/XamlControlsGallery.DesktopWap.Package.wapproj

```diff
@@ -42,8 +42,8 @@
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <DefaultLanguage>en-US</DefaultLanguage>
     <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
-    <EntryPointProjectUniqueName>XamlControlsGallery.DesktopWap.csproj</EntryPointProjectUniqueName>
-    <AppxBundleNameForOutput>XamlControlsGallery.DesktopWap</AppxBundleNameForOutput>
+    <EntryPointProjectUniqueName>WinUI3Gallery.DesktopWap.csproj</EntryPointProjectUniqueName>
```

Comment:
Note: This change prevents us from building & running the solution until we rename the project file